wafer/templates/wafer/nav.html

```diff
@@ -14,7 +14,7 @@
     <a class="navbar-brand" href="{% url 'wafer_page' %}">{{ WAFER_CONFERENCE_NAME }}</a>
     {% endblock navtitle %}
 
-    <div class="collapse navbar-toggleable-xs pull-xs-right" id="wafer-navbar-collapse">
+    <div class="collapse navbar-toggleable-xs" id="wafer-navbar-collapse">
```

Comment:
Just checking whether removing pull-xs-right entirely rather than replacing it with float-xs-right is the correct thing to do here.